mesosphere/kudo-cassandra-operator
# Parameterize Docker image and pull policies for all images used

## What changes are proposed in this PR? 
Resolves  [DCOS-60288](https://jira.mesosphere.com/browse/DCOS-60288)

This PR Implement adding:

- **DOCKER_IMAGE**  to `params.yaml`
- **IMAGE_PULL_POLICY** to  `params.yaml`

## How were these changes tested?
Manually tested  
1. Install operator 
2. Check stateful-set configuration
